### PR TITLE
Overhaul

### PR DESCRIPTION
**Pull request by lryanle**

### CODE REVIEW

File: src/app/components/NavBar.tsx
Comment (lryanle) [inline src/app/components/NavBar.tsx:1]: remove this component

File: src/app/components/OrganizingTeam.tsx
Comment (lryanle) [inline src/app/components/OrganizingTeam.tsx:1]: move to `/src/components`

File: src/app/page.tsx [hunk 1]
```diff
@@ -15,8 +16,10 @@ export default function Home() {
           <div id="faqSection">
             <FAQ />
           </div>
-
-          <Venue />
+          <div id="venueSection">
+            <Venue></Venue>
```
Comment (lryanle) [inline src/app/page.tsx]: `<Venue></Venue>` -> `<Venue />`

File: src/app/page.tsx [hunk 2]
```diff
@@ -15,8 +16,10 @@ export default function Home() {
           <div id="faqSection">
             <FAQ />
           </div>
-
-          <Venue />
+          <div id="venueSection">
```
Comment (lryanle) [inline src/app/page.tsx:19]: what is `id` for?

File: src/components/Hero.tsx
Comment (lryanle) [inline src/components/Hero.tsx:1]: keep this button removed

File: src/app/globals.css
Comment (lryanle) [inline src/app/globals.css:1]: There are rebase fragments here

File: src/app/globals.css [hunk 3]
```diff
@@ -2,21 +2,54 @@
...
+    --accent-foreground: 0 0% 98%;
+    --destructive: 0 62.8% 30.6%;
+    --destructive-foreground: 0 0% 98%;
+    --border: 240 3.7% 15.9%;
+    --input: 240 3.7% 15.9%;
+    --ring: 240 4.9% 83.9%;
   }
 }
 
+<<<<<<< HEAD
```
Comment (lryanle) [inline src/app/globals.css]: rebase fragment

File: src/app/globals.css [hunk 4]
```diff
@@ -30,6 +63,15 @@ body {
       rgb(var(--background-end-rgb))
     )
     rgb(var(--background-start-rgb));
+=======
```
Comment (lryanle) [inline src/app/globals.css]: rebase fragment

File: src/app/globals.css [hunk 5]
```diff
@@ -30,6 +63,15 @@ body {
...
+@layer base {
+  * {
+    @apply border-border dark;
+  }
+  body {
+    @apply bg-background text-foreground;
+  }
+>>>>>>> ab80948b93c4eeaa42e250c67d61405d0a962128
```
Comment (lryanle) [inline src/app/globals.css]: rebase fragment

File: src/app/globals.css (hunk 3 above)
Comment (lryanle) [reply src/app/globals.css]: Remove everything incoming from head.

File: src/components/Hero.tsx [hunk 6]
```diff
@@ -4,6 +4,7 @@ import { PulseBeams } from "./ui/pulse-beams";
 import Link from "next/link";
 
 export default function Homepage() {
+<<<<<<< HEAD
```
Comment (lryanle) [inline src/components/Hero.tsx]: fragment

File: src/components/Hero.tsx [hunk 7]
```diff
@@ -52,4 +53,58 @@ export default function Homepage() {
       </div>
     </div>
   );
+=======
```
Comment (lryanle) [inline src/components/Hero.tsx]: fragment

File: src/components/Hero.tsx [hunk 8]
```diff
@@ -52,4 +53,58 @@ export default function Homepage() {
...
+									></path>
+								</g>
+							</svg>
+						</a>
+					</div>
+				</div>
+			</div>
+		</div>
+	);
+>>>>>>> ab80948b93c4eeaa42e250c67d61405d0a962128
```
Comment (lryanle) [inline src/components/Hero.tsx]: fragment

File: src/app/page.tsx (hunk 2 above)
Comment (Nishchit-Dev) [reply src/app/page.tsx:19]: so the id were used to scroll to that section
but the buttons are removed i guess they wont be needed

File: src/components/Hero.tsx
Comment (Nishchit-Dev) [reply src/components/Hero.tsx:1]: come again please.

**Comment by lryanle** [comment]
yolo

**Pull request merged**